ArangoDB Setup.json

```diff
@@ -672,13 +557,13 @@
 							}
 						},
 						"url": {
-							"raw": "{{arangoUrl}}/_db/networkmap/_api/collection",
+							"raw": "{{arangoUrl}}/_db/configuration/_api/collection",
```

Comment:
These strings exist in the environment files as environment variables, e.g. `{{db_config_all}}` for `configuration` - these values have been updated in the `tms-configuration` collections as well, for reference. Can we please update these hard-coded database and collection names to use the environment variables?